src/appleseed/renderer/modeling/bsdf/glassbsdf.cpp

```diff
@@ -88,6 +88,8 @@ namespace
     class GlassBSDFImpl
       : public BSDF
     {
+        typedef GlassBSDFInputValues InputValues;
```

Comment:
Move to `private` section at the end of the class.

Reply:
I tried, but gcc 4.8 does not compile if I move the line down.

Reply:
I tried, but gcc 4.8 does not compile if I move the line down.

Reply:
Interesting.